specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/sqlX.json

```diff
@@ -0,0 +1,261 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CosmosDBServiceResourceCreate": {
+            "$ref": "./examples/CosmosDBSqlAcceleratorCreate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SqlAcceleratorService"
+        ],
+        "description": "Deletes the sql accelerator Service for a Cosmos DB account.",
+        "operationId": "SqlAccelerators_Delete",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The sqlX delete operation will complete asynchronously."
+          },
+          "204": {
+            "description": "SqlX service does not exist in the database account."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/DefaultErrorResponse"
+            }
+          },
+          "x-ms-examples": {
+            "CosmosDBServiceResourceDelete": {
+              "$ref": "./examples/CosmosDBSqlAcceleratorDelete.json"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServiceResourceCreateUpdateParameters": {
+      "description": "Parameters to create a service resource",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Properties to create and update an Azure Cosmos DB Service Resource.",
+          "$ref": "#/definitions/ServiceResource",
+          "x-ms-client-flatten": true
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "ServiceGetResults": {
+      "description": "An Azure Cosmos DB Cassandra table.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties of an Azure Cosmos DB Cassandra table",
+          "$ref": "#/definitions/ServiceResource"
+        }
+      }
+    },
+    "ServiceResource": {
+      "description": "A CosmosDB Service resource",
+      "type": "object",
+      "properties": {
+        "serviceType": {
+          "description": "Specifies the type of Service.",
+          "type": "string",
+          "enum": [
+            "Sqlx"
+          ]
+        },
+        "status": {
+          "description": "Status of the CosmosDB Service.",
```

Comment:
Does removing the word "CosmosDB" makes sense?

Reply:
Most other docs have stuff like "Azure CosmosDB sql Database", I think there is no harm in having a description this detailed.